scrape_docs.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+import click
+from bs4 import BeautifulSoup
```

Comment:
Hmm, requirements.txt is more for the programmers I think, we could add it there even though someone wouldn't want to use it because not working on a very new api, it doesn't really hurt to install it anyway in their dev-environment.

The enduser's docs (if they install from git) are handled within setup.py. We should definitely NOT add it there in my opinion.